app/routes/_index.tsx

```diff
@@ -45,7 +45,7 @@ export default function App() {
         <WidgetStampy />
 
         <div className="padding-bottom-80" />
-        <h3 className="grey large-bold padding-bottom-32">Advanced content</h3>
+        <h3 className="grey padding-bottom-32">Advanced content</h3>
```

Comment:
Heh no so, we want it to be a p tag, I'm not using h3s.

`<p className="grey large-bold padding-bottom-32">Advanced content</p>`

Reply:
@melissasamworth  Thanks for the note. I changed the tag to a `p`.

Should it still have the padding-bottom-32 class? How can one get the desired padding from the figma?